projects/packages/status/src/class-errors.php

```diff
@@ -19,11 +19,13 @@ class Errors {
 	/**
 	 * Catches PHP errors.  Must be used in conjunction with output buffering.
 	 *
+	 * @deprecated since $$next-version$$
```

Comment:
Wouldn't hurt to stick `@deprecated` in the class comment too, just to be extra clear about it.